biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -20,6 +20,8 @@ module EDMortalityFunctionsMod
    use EDParamsMod           , only : fates_mortality_disturbance_fraction
    use FatesInterfaceMod     , only : bc_in_type
 
+   use PRTGenericMod,          only : carbon12_species
```

Comment:
why is is 'carbon12_species' here and 'all_carbon_species' in the logging mortality function?

Reply:
This has been a tricky bit. I tried to make it so PARTEH would be ready to handle carbon isotopes. However, there are no specific PARTEH methods that use isotopic carbon. And as we all know, there aren't any methods in FATES master that use isotopic carbon.  carbon12 indicates that the state variable of interest is just the single species of carbon, carbon 12; whereas all_carbon_species is a flag that specifies the sum of all carbon isotopes for the organ of interest should be returned.  Since we only have carbon12 used in current hypotheses, the two are currently equivalent.  

The process of deciding which of these two should be passed between different parts of the code has been a little murky, and this might be a sign of me not being able to make up my mind or finish thinking it through. 

In the example you bring up @rosiealice , I agree these should definitely be the same.

I suppose that as a default, we could just use "all_carbon_species" to specify all carbon fluxes in and out of PARTEH until an isotope model is introduced.